src/optimization/HolonomicTrajoptUtil.inc

```diff
@@ -47,6 +51,21 @@ void ApplyHolonomicConstraint(Opti& opti, const Expr& x, const Expr& y,
   } else if (std::holds_alternative<LinePointConstraint>(holonomicConstraint)) {
     ApplyConstraint(opti, x, y, theta,
                     std::get<LinePointConstraint>(holonomicConstraint));
+  } else if (std::holds_alternative<PointAtConstraint>(
+                 holonomicConstraint)) {
+    auto pointAtConstraint = std::get<PointAtConstraint>(holonomicConstraint);
+    double fieldPointX = pointAtConstraint.fieldPointX;
+    double fieldPointY = pointAtConstraint.fieldPointY;
+    auto headingBound = pointAtConstraint.headingTolerance;
+    auto dx = fieldPointX - x;
+    auto dy = fieldPointY - y;
+    auto hx = cos(theta);
+    auto hy = sin(theta);
+    auto ux = dx / sqrt(dx*dx + dy*dy);
+    auto uy = dy / sqrt(dx*dx + dy*dy);
```

Comment:
You can use hypot(dx, dy) here instead of sqrt(dx² + dy²), which has a more efficient and numerically stable autodiff implementation.